src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -66,47 +67,38 @@ class CompanyStep extends React.Component {
      */
     validate() {
         if (!this.state.password.trim()) {
-            Growl.error(this.props.translate('common.passwordCannotBeBlank'));
```

Comment:
Just a quick question since I'm not really up to speed on this... 

In this PR, we are replacing all these calls with red outlines and inline messages -> https://github.com/Expensify/App/pull/4431

Do we need both a confirm modal and the inline messages?

Reply:
That's what I gathered from the solution mentioned in the [issue](https://github.com/Expensify/Expensify/issues/171666#issue-951712947). 
> Specific fields causing errors will be highlighted thanks to this change so we can use a generic message in the modal for all cases:
"Oops something went wrong! Please double check any highlighted fields and try again."

We can still wait for @MitchExpensify to confirm.

Reply:
Yes we're replacing all growls with centered modals as well as adding red field outlines.

Reply:
👍